RestApi/Sepes.RestApi/Services/SepesDb.cs

```diff
@@ -102,28 +135,27 @@ private static void createInsertValues(int studyId, int[] array, StringBuilder s
 
             Console.WriteLine(strBuilder.ToString());
         }
-        /*public Task<int> updateStudy(Study study)
+
+        public async Task<int> updateStudy(Study study)
```

Comment:
In the recent refactoring we went away from passing the full models into functions. Its intended to be written in the format: UpdateStudy(String studyId, bool archived) so its more readable when we use the function.